lib/Cargo.toml

```diff
@@ -75,7 +75,7 @@ jni = { version = "0.21.1", features = ["invocation"] }
 paranoid-android = "0.2.1"
 
 [features]
-default = ["use_ffmpeg", "use_livekit", "use_deno"]
+default = ["use_ffmpeg", "use_livekit", "use_deno", "enable_inspector"]
```

Comment:
It's ok that it's going to be able for Desktop? I think makes sense, because we don't publish Desktop.